packages/shared/components/Idle.svelte

```diff
@@ -20,11 +20,9 @@
 
             const ap = get(activeProfile)
             if (ap) {
-                const timeoutDuration =
-                    MILLISECONDS_PER_SECOND * SECONDS_PER_MINUTE * ap.settings.lockScreenTimeout
+                const timeoutDuration = MILLISECONDS_PER_SECOND * SECONDS_PER_MINUTE * ap.settings.lockScreenTimeout
 
-                if(!isIdleTimeValid(new Date(), timeoutDuration))
-                    lock()
+                if (!isIdleTimeValid(new Date(), timeoutDuration)) lock()
```

Comment:
While the conventions in the wiki don't explicitly say so, our current usages of `if` statements (as well as the examples in the wiki) imply that this should use braces and be on 3 lines:

```ts
if (!isIdleTimeValid(new Date(), timeoutDuration)) {
  lock()
}
```

For example:

https://github.com/iotaledger/firefly/blob/d62300a7fe307e6f158774c43cfada8bc5a6b307/packages/shared/lib/actions.ts#L21-L23

Based on [this SO answer](https://stackoverflow.com/a/65721310/3188334) it looks like `prettier` will do this unless we put curly braces, but maybe we can enforce this through the [`curly` rule](https://eslint.org/docs/rules/curly) for ESLint

Reply:
Recently I've been formalizing our coding conventions along with some new ones into the wiki. I'll update the [brackets section](https://github.com/iotaledger/firefly/wiki/Guide:-Coding-Conventions#brackets) with some rules for braces specifically.

I think adding the `curly` rule is a good idea :+1:, it can be part of the shared library refactor!

Reply:
I agree! Should I enable those rules and then merge or merge this first? I'm more inclined to the latter option, since that would not add extra scope to this already massive PR. Will merge if @rajivshah3 approves :)

Reply:
I would merge this first since it is slightly out of scope of this PR. We can just include it in the library refactor :+1: